From 91fb88c1823765b03c67a87e9523abafd06e9e6c Mon Sep 17 00:00:00 2001
From: Jeegar Patel <jeegar.patel@xilinx.com>
Date: Fri, 25 Sep 2020 02:11:16 -0700
Subject: [PATCH 1673/1851] drm: xlnx: pl_disp: Fix MIXED_ENUM_TYPE type
 warning

commit 6655151e3a6ec70d892c2dc7b0306f1ee938b3d3 from
https://github.com/Xilinx/linux-xlnx.git

This patch fix MIXED_ENUM_TYPE type coverity warning.

Event mixed_enum_type: 	enumerated type mixed with another type.

Signed-off-by: Jeegar Patel <jeegar.patel@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_pl_disp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_pl_disp.c b/drivers/gpu/drm/xlnx/xlnx_pl_disp.c
index d8e7a9f22779..db720ac5f19c 100644
--- a/drivers/gpu/drm/xlnx/xlnx_pl_disp.c
+++ b/drivers/gpu/drm/xlnx/xlnx_pl_disp.c
@@ -167,7 +167,7 @@ static void xlnx_pl_disp_plane_enable(struct drm_plane *plane)
 {
 	struct xlnx_pl_disp *xlnx_pl_disp = plane_to_dma(plane);
 	struct dma_async_tx_descriptor *desc;
-	enum dma_ctrl_flags flags;
+	unsigned long flags;
 	struct xlnx_dma_chan *xlnx_dma_chan = xlnx_pl_disp->chan;
 	struct dma_chan *dma_chan = xlnx_dma_chan->dma_chan;
 	struct dma_interleaved_template *xt = &xlnx_dma_chan->xt;
-- 
2.31.1

